rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.adoc

```diff
@@ -318,12 +318,16 @@ Type::
 
 | `protocol`
 | `string`
-| protocol specifies whether the IngressController expects incoming connections to use plain TCP or whether the IngressController expects PROXY protocol. 
- PROXY protocol can be used with load balancers that support it to communicate the source addresses of client connections when forwarding those connections to the IngressController.  Using PROXY protocol enables the IngressController to report those source addresses instead of reporting the load balancer's address in HTTP headers and logs.  Note that enabling PROXY protocol on the IngressController will cause connections to fail if you are not using a load balancer that uses PROXY protocol to forward connections to the IngressController.  See http://www.haproxy.org/download/2.2/doc/proxy-protocol.txt for information about PROXY protocol. 
- The following values are valid for this field: 
- * The empty string. * "TCP". * "PROXY". 
+| protocol specifies whether the IngressController expects incoming connections to use plain TCP or whether the IngressController expects PROXY protocol.
+ PROXY protocol can be used with load balancers that support it to communicate the source addresses of client connections when forwarding those connections to the IngressController.  Using PROXY protocol enables the IngressController to report those source addresses instead of reporting the load balancer's address in HTTP headers and logs.  Note that enabling PROXY protocol on the IngressController will cause connections to fail if you are not using a load balancer that uses PROXY protocol to forward connections to the IngressController.  See http://www.haproxy.org/download/2.2/doc/proxy-protocol.txt for information about PROXY protocol.
+ The following values are valid for this field:
+ * The empty string. * "TCP". * "PROXY".
  The empty string specifies the default, which is TCP without PROXY protocol.  Note that the default is subject to change.
 
+ | `hostNetwork`
+ | `integer`
+ | hostNetwork is used to set binding ports for the Ingress Controller so that multiple Ingress Controllers can be deployed on the same node. The following optional node ports with default values can be used; `httpPort: 80`, `httpsPort: 443`, and `statsPort: 1936`. The range of the priority value can go from positive to negative in the range of int32.
```

Comment:
```suggestion
 | `httpPort`
 | `integer`
 | HTTPPort is the port on the host which should be used to listen for HTTP requests. Define this field when default port 80 is known to be used by another process. Please avoid specifying a port within the nodeport range, which defaults to 30000-32767, but could be configured differently.
  | `httpsPort`
 | `integer`
 | HTTPSPort is the port on the host which should be used to listen for HTTPS requests. Define this field when default port 443 is known to be used by another process. Please avoid specifying a port within the nodeport range, which defaults to 30000-32767, but could be configured differently.
  | `statsPort`
 | `integer`
 | StatsPort is the port on the host where the stats from the router are published. Please avoid specifying a port within the nodeport range, which defaults to 30000-32767, but could be configured differently.
f you configure an external load-balancer to forward connections this ingress controller, the load balancer should use this port for health checks.  A load balancer can send HTTP probes to this port on a given  node with the path /healthz/ready to determine whether the  ingress controller is ready to receive traffic on the node. For proper operation, the load balancer must not forward traffic  to a node until /healthz/ready reports ready, and the load balancer must stop forwarding within a maximum of 45 seconds after /healthz/ready starts reporting not-ready.  Probing every 5 or 10 seconds, with a 5-second timeout and with a threshold of two successful or failed requests to become healthy or unhealthy, respectively, are well-tested values.
```

Reply:
@Amrita42 this table is already describing the fields in hostNetwork. 
If you want to add a description for hostNetwork itself, you could eventually put it line 306

> hostNetwork is used to set binding ports for the Ingress Controller so that multiple Ingress Controllers can be deployed on the same node. The following optional node ports with default values can be used; `httpPort: 80`, `httpsPort: 443`, and `statsPort: 1936`. The range of the priority value can go from positive to negative in the range of int32.

Reply:
Probing every 5 or 10 seconds, with a 5-second timeout and with a threshold of two successful or failed requests to become healthy or unhealthy, respectively, are well-tested values. >> I am not sure if we need to retain this statement , I have not added it wdyt?

Reply:
Probably... 
Let's bring it to @Miciah 's attention to make sure. 
To be honest, I simply copied and pasted all that we wanted to put in the api doc .